cosmosdb-scaler/README.md

```diff
@@ -0,0 +1,42 @@
+# KEDA External Scaler for Azure Cosmos DB
+
+Chart for installing KEDA external scaler for Azure Cosmos DB.
+
+- [Documentation](https://github.com/kedacore/keda-external-scaler-azure-cosmos-db#readme)
+- [Release Notes](https://github.com/kedacore/keda-external-scaler-azure-cosmos-db/releases/tag/v0.1.0)
+- [Example Usage](https://github.com/kedacore/keda-external-scaler-azure-cosmos-db/tree/main/src/Scaler.Demo)
+
+## Installation
+
+1. Add and update Helm chart repo.
+
+    ```shell
+    helm repo add kedacore https://kedacore.github.io/charts
+    helm repo update
+    ```
+
+1. Install KEDA Helm chart (*or follow one of the other installation methods on [KEDA documentation](https://keda.sh/docs/deploy)*).
+
+    ```shell
+    helm install keda kedacore/keda --namespace keda --create-namespace
+    ```
+
+1. Install 'Cosmos DB scaler' Helm chart.
+
+    ```shell
+    helm install cosmosdb-scaler kedacore/keda-external-scaler-azure-cosmos-db --namespace keda --create-namespace
```

Comment:
This is not correct, we should align `keda-external-scaler-azure-cosmos-db` with the new folder name